src/figuresClasses.ts

```diff
@@ -1,19 +1,84 @@
+
+enum Shape {
+  Triangle = 'triangle',
+  Rectangle = 'rectangle',
+  Circle = 'circle',
+}
+
 export interface Figure {
+  shape: Shape;
+  color: 'red' | 'green' | 'blue',
 
+  getArea: () => number;
 }
 
-export class Triangle {
+export class Triangle implements Figure {
+  public shape = Shape.Triangle;
+
+  constructor(
+    public color: 'red' | 'green' | 'blue',
+    public a: number,
+    public b: number,
+    public c: number,
+  ) {
+    if (a <= 0 || b <= 0 || c <= 0) {
+      throw new Error('wrong length specified');
```

Comment:
<img width="1043" alt="image" src="https://github.com/mate-academy/ts_figures_classes/assets/86789161/6dfb9112-e3e7-46d1-9bca-7490fbd2182d">

Same comment to all other error messages